numba/cgutils.py

```diff
@@ -659,14 +659,24 @@ def unpack_tuple(builder, tup, count=None):
 
 def get_item_pointer(context, builder, aryty, ary, inds, wraparound=False,
                      boundscheck=False):
+    ptr, _, _ = get_item_array_pointer(
+        context, builder, aryty, ary, inds,
+        wraparound=wraparound, boundscheck=boundscheck
+    )
+    return ptr
+
+
+def get_item_array_pointer(context, builder, aryty, ary, inds, wraparound=False,
+                           boundscheck=False):
     # Set boundscheck=True for any pointer access that should be
     # boundschecked. do_boundscheck() will handle enabling or disabling the
     # actual boundschecking based on the user config.
     shapes = unpack_tuple(builder, ary.shape, count=aryty.ndim)
     strides = unpack_tuple(builder, ary.strides, count=aryty.ndim)
```

Comment:
Seemed a shame to compute these only to throw them away again, hence the new function rather than just changing the caller..